central/cloudsources/manager/manager_test.go

```diff
@@ -113,8 +140,63 @@ func TestMatchDiscoveredClusters(t *testing.T) {
 	}
 }
 
+func TestChangeDiscoveredClustersStatus(t *testing.T) {
```

Comment:
How about extracting this to a separated file as this is the only test here that require PG